src/proteus/utils/coupler.py

```diff
@@ -432,7 +433,7 @@ def UpdatePlots( hf_all:pd.DataFrame, dirs:dict, config:Config, end=False, num_s
         nc_times = [int(f.split("/")[-1].split("_atm")[0]) for f in ncs]
 
         # Check intersection of atmosphere and interior data
-        if not spider:
+        if dummy_int:
```

Comment:
At next iteration I would suggest to make this maximally clear:
if dummy_int:
elif: spider
elif: aragog
else: throw error